### DISCUSSION TITLE
Python version

### DISCUSSION THREAD

**Discussion by nate-norris** [semuconsulting/pynmeagps#40] [General]
Hello, I have AirMar weather stations that use NMEA 0183 coming in soon so I have not been able to test this but I was curious if there were any known reasons why Python 3.7 would have issues. I see only 3.8 and above listed. Thank you!

**Comment by nate-norris** [comment]
I just saw 3.7 was removed from CONTRIBUTING.md commit so looks like 3.7 has confirmed issues. Thanks.

**Comment by semuadmin** [comment]
Hi @jonathan-s-norris 

Python 3.7 is now end of life (as of 27/06/2023 - see https://devguide.python.org/versions/), but that's the only reason it's been removed from the supported list. I'm not actually aware of any issues with 3.7 at this point, though it will inevitably be dropped from more and more repos as time goes on.

If you have a 3.7 machine and want to give it a go, I'm happy to try and invesigate any issues you might have, but I can't guarantee I'll be able to fix them!

**Reply by nate-norris** [reply]
Wow thanks for the quick reply. That makes sense. I'm going to be maintaining two different versions of an application with the original having BLE libraries that have had issues with anything above 3.7. It is not that big of a problem to change the python version if I decide to run your library but I'll go ahead and see how it runs. Thanks again.
